esp-radio/src/wifi/mod.rs

```diff
@@ -3022,7 +3022,7 @@ impl WifiController<'_> {
     }
 
     /// A blocking wifi network scan with caller-provided scanning options.
-    pub fn scan_with_config_sync(
+    pub fn scan_with_config(
```

Comment:
We never use the `_sync` (or even `_blocking`) suffix, not sure why we did here.

Reply:
I think we removed most of these a while ago, I guess this one stuck around.